ios/RNSVVideoPlayer.mm

```diff
@@ -34,7 +34,8 @@ - (instancetype)initWithURL:(NSURL*)url
   _player.actionAtItemEnd = AVPlayerActionAtItemEndNone;
```

Comment:
The _player is not initialized when you use this method. Are we sure it is working as supposed ?

initialisation line 51 ( after)